cmd/minishift/cmd/start_preflight.go

```diff
@@ -467,17 +467,23 @@ func isMounted(driver drivers.Driver, mountpoint string) (bool, error) {
 }
 
 // checkIsoUrl checks the Iso url and returns true if the iso file exists
-func checkIsoUrl() bool {
+func checkIsoURL() bool {
 	isoUrl := viper.GetString(configCmd.ISOUrl.Name)
 	for _, isoAlias := range constants.ValidIsoAliases {
 		if isoUrl == isoAlias {
 			return true
 		}
 	}
+
 	if !strings.HasSuffix(isoUrl, ".iso") {
 		return false
 	}
 
+	validScheme := regexp.MustCompile(`^https?://`)
+	if validScheme.MatchString(isoUrl) {
```

Comment:
```
if validScheme := regexp.MustCompile(`^https?://`); validScheme.MatchString(isoUrl) {
   return true
}
```

Reply:
If you just want to match a string regexp.MatchString is probably a better choice. See also https://gobyexample.com/regular-expressions

Reply:
`regexp.MatchString` is what used in the following string matches. I do not know the advantage or disadvantage of one over the other. this does the job in two lines.

Reply:
`regexp.MatchString` works for me. It's ideal at this point.

`With MatchString, we see if a pattern can match a string. This tells us if the string satisfies the pattern's requirements. We use Compile to create reusable objects.` from  here https://www.dotnetperls.com/regexp-go

Reply:
I think `regexp.MatchString` should be used here, Thanks @hferentschik for the pointer to that.